### PR TITLE
Add timeseries data complexity

### PR DESCRIPTION
**Pull request by spiderjako**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1256,6 +1257,25 @@ Example result:
 
 ```
 
+### Metric Complexity
+
+Fetch the complexity of a query
```
Comment (IvanIvanoff) [inline README.md:1262]:
```suggestion
Fetch the complexity of a metric. The complexity depends on the from/to/interval parameters, as well as the metric and the subscription plan. A request might have a maximum complexity of 20000. If a request has a higher complexity there are a few ways to solve the issue:
- Break down the request into multiple requests with smaller from-to ranges.
- Upgrade to a higher subscription plan.



```

File: README.md [hunk 2]
```diff
@@ -795,6 +796,25 @@ datetime                              slug     change    status
...
+
+```python
+san.metric_complexity(
+    metric='price_usd',
+    from_datetime='2020-02-02',
```
Comment (tspenov) [inline README.md:806]:
```suggestion
    from_date='2020-02-01',
```

File: README.md [hunk 3]
```diff
@@ -795,6 +796,25 @@ datetime                              slug     change    status
...
+    to_datetime='2020-03-20',
```
Comment (tspenov) [inline README.md:807]:
```suggestion
    to_date='2020-02-20',
```

File: README.md [hunk 4]
```diff
@@ -795,6 +796,25 @@ datetime                              slug     change    status
...
+    from_date='2020-02-01',
+    to_date='2020-02-20',
+    interval='1d'
+)
+```
+
+Example result:
+
+```
+29
```
Comment (tspenov) [inline README.md:815]: Also the result should be different.

File: setup.py [hunk 5]
```diff
@@ -6,7 +6,7 @@
 
 setuptools.setup(
     name="sanpy",
-    version="0.7.19",
+    version="0.7.20",
```
Comment (tspenov) [inline setup.py:9]: I suggest bumping to `0.8.0`

**Pull request merged**